scipy/interpolate/src/_fitpackmodule.c

```diff
@@ -191,14 +188,17 @@ fitpack_bispev(PyObject *dummy, PyObject *args)
     ny = PyArray_DIMS(ap_ty)[0];
     mx = PyArray_DIMS(ap_x)[0];
     my = PyArray_DIMS(ap_y)[0];
-    mxy = mx*my;
-    if (my != 0 && mxy/my != mx) {
+   /* v = INT_MAX/my is largest integer multiple of `my` such that
+       v * my <= INT_MAX
+    */
+    if (my != 0 && F_INT_MAX/my < mx) {
```

Comment:
```suggestion
    if (my != 0 && int_max/my < mx) {
```

Reply:
What is `int_max` here, in code, was it meant to be `F_INT_MAX`?